CONTRIBUTING.md

```diff
@@ -3,6 +3,12 @@
 We welcome community contributions to this library and we hope that together we can expand the coverage of ASM ready data for everyone.
 
 In order to contribute you will need to have an Individual or Corporate Contributor License Agreement (CLA) on file with Benchling depending on if you are contributing on your own time or as part of another company. When you make your first pull request we will check if you have a CLA and if not take care of that with you first. The process is quick and painless and helps us to make sure that you and everyone who uses your code in the future is protected.
+## GPG keys and signed commits
+This repository uses signed commits. Please follow these links to get set up.
```

Comment:
could you change this to something like:

`All commits to this repository must be signed. To set up commit signatures, please do the following:`